react/src/components/search-app/SearchResultsComponent.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+
+export default class SearchResultsComponent extends React.Component {
```

Comment:
Haha, I think we are again in the discussion that we had on the very first day :D Do we want the front-end components to be as autonomous as possible (fetch data per API call), or let Express do part of the rendering.

The second way could (maybe) be more performant, but I believe that the first approach will allow us to include the component into other sites more simply.